src/drivers/gpio/imx/imx.c

```diff
@@ -35,7 +33,7 @@ EMBOX_UNIT_INIT(imx_gpio_init);
 static struct gpio_chip imx_gpio_chip;
 
 static int imx_gpio_init(void) {
-	for (int i = 0; i <= GPIO_MAX; i++)
+	for (int i = 0; i <= GPIO_PORTS; i++)
```

Comment:
for (int i = 0; i <= GPIO_PORTS; i++) {
}
Braces?

Reply:
Fixed.